src/routers/confirmCompanyRouter.ts

```diff
@@ -13,6 +15,14 @@ router.get("/", handleExceptions(async (req: Request, res: Response, _next: Next
 }));
 
 router.post("/", handleExceptions(async (req: Request, res: Response, _next: NextFunction) => {
+
+    const session = req.session as Session;
+    const companyNumber = req.query.companyNumber as string;
+    const DESCRIPTION = "PSC Verification Transaction";
+    const REFERENCE = "PscVerificationReference";
+
+    const transaction: Transaction = await postTransaction(session, companyNumber, DESCRIPTION, REFERENCE);
```

Comment:
in the companyProfile service, the request is passed to it, and then it pulls out the session and company number.  I wonder if we should be consistent, whichever way we go.
Plus something doesn't seem quite appropriate about passing the DESCRIPTION and REFERENCE.  I don't know the best place, but would just having them in the transaction service for now be neater?